wfd/public/video_format.h

```diff
@@ -147,6 +150,25 @@ struct H264VideoFormat {
   RateAndResolution rate_resolution;
 };
 
+struct SupportedH264VideoFormats {
```

Comment:
could you pls explain the idea of SupportedH264VideoFormats vs SelectableH264VideoFormat (let's say vector<SelectableH264VideoFormat>)?